### DISCUSSION TITLE
help understanding about 'pre'

### DISCUSSION THREAD

**Discussion by JustBennnn** [pydantic/pydantic#3351] [Question] [answered]
Hi,
So I was just reading through the validation section  in the pydantic documentation, and it states that 'the keyword argument pre will cause the validator to be called prior to other validation'. Please could someone explain what is meant by 'other validation' and how do I know when to use 'pre'? 
Thanks

**Comment by Kludex** [comment, marked as answer]
We don't have the `pre` anymore in Pydantic V2. We are using `mode="before"` instead.

See more on the [Model validators](https://docs.pydantic.dev/latest/usage/validators/#model-validators) section.
